indico/modules/categories/controllers/management.py

```diff
@@ -512,3 +513,8 @@ def _process(self):
                 self.role.members.add(user)
                 logger.info('User %r added to role %r by %r', user, self.role, session.user)
         return jsonify_data(html=_render_role(self.role, collapsed=False))
+
+
+class RHAddCategoryRoleMembersImportCSV(ImportRoleMembersMixin, RHManageCategoryRole):
+    """Add users to a category role from CSV"""
+    pass
```

Comment:
the `pass` isn't needed since the docstring already make the class' body non-empty